lms/templates/admin/application_instance/show.html.jinja2

```diff
@@ -101,8 +104,12 @@
                                 <input type="submit" class="button is-primary" value="Save" />
                             </div>
                         </fieldset>
-                    </li>
-                    <li>
+                    </form>
+                </li>
+                <li>
+                    <form method="POST"
+                          action="{{ request.route_url("admin.instance", id_=instance.id) }}">
```

Comment:
Aren't these nested quotes problematic? Should we do `'admin.instance'` instead of `"admin.instance"` just in case?

Reply:
The ones inside `{{` are python, they ignore the rest of the HTML. 

I don't have a strong preference but our python formatter prefers `"` in general so keeping these.

Reply:
> The ones inside {{ are python, they ignore the rest of the HTML.

I guessed that could perfectly be the case.